src/front/glsl/lex.rs

```diff
@@ -76,6 +76,9 @@ impl<'a> Iterator for Lexer<'a> {
                     "highp" => TokenValue::PrecisionQualifier(Precision::High),
                     "mediump" => TokenValue::PrecisionQualifier(Precision::Medium),
                     "lowp" => TokenValue::PrecisionQualifier(Precision::Low),
+                    "restrict" => TokenValue::Restrict,
+                    "readonly" => TokenValue::ReadOnly,
```

Comment:
are we just ignoring them?

Reply:
I had the logic in a separate branch, I've added the commit.